great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -3724,3 +3756,52 @@ def escape_all_config_variables(
                 return value.replace("$", dollar_sign_escape_string)
             return value
         return value.replace("$", dollar_sign_escape_string)
+
+    def save_config_variable(
```

Comment:
This method has some pretty filesystem specific logic but a larger overhaul to file-based logic will occur shortly.

In short, both `FileDataContext` and `CloudDataContext` have "fileystem-enabled" capabilities - these two classes should inherit from a separate `_FilesystemEnabledDataContext` or something to that effect.

For now, I think it's okay to put in `AbstractDataContext`.